src/styles/card.scss

```
@@ -0,0 +1,88 @@
+body {
+  font-family: "Roboto", sans-serif;
```

You have this `fontRoboto` variable, use it here
```suggestion
  font-family: $fontRoboto;
```